src/demo/resources/demo/shop/shop-postgres-mongodb.ben.xml

```diff
@@ -59,7 +59,6 @@
     <comment>define a mongodb database that will be referred by the id 'mongodb' subsequently</comment>
     <mongodb id="mongodb" host="{ftl:${mongoHost}}" port="{ftl:${mongoPort}}" database="{ftl:${mongoDatabase}}"
              user="{ftl:${mongoUser}}" password="{ftl:${mongoPassword}}"
-             authenticationDatabase="{ftl:${mongoAuthenticationDatabase}}"/>
```

Comment:
here the same , doesn't seem to have to do anything with the csv error

Reply:
Yes, it doesn't have relative to the csv error. I just see the code wrong went having 2 close tag /> so it change it.

Reply:
good point alright